### PR TITLE
Add Mobile menu

### PR DESCRIPTION
**Pull request by omer-creator**
*Add a mobile menu for the portfolio*
> Use HTML and CSS best practices.
> Use JavaScript Best practices.
> add eslinrs.json file for JS linters.

### CODE REVIEW

**Review by nahnahdev — COMMENTED** [review]
## Project Status: INVALIDATED :red_circle: 
Hi @omer-creator and @AbenezerAssefa , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that is missing the `Eslint` checks. For this milestone, please ensure that you set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository.
- You should have 5 linter checks in all including the `Eslint` checks instead of 4.
- Your linter checks
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by abdulhamiid — COMMENTED** [review]
### Hi @omer-creator, @AbenezerAssefa 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ ❌ 

As requested by the previous reviewer, for this milestone, please ensure that you set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository.

Expected Checks (5)
image

Your Checks (4)
image

You have submitted a project, that <EXPLAIN WHY PROJECT IS INVALID>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -30,8 +30,20 @@
...
+      <div class="mobile-menu">
+        <div class="mobile-menu-header">
+        </div>
+        <ul class="mobile-menu-items">
+          <li><a href="#portfolio" class="mobile-header-color">Portfolio</a></li>
+          <li><a href="#about" class="mobile-header-color">About</a></li>
+          <li><a href="#contact" class="mobile-header-color">Contact</a></li>
+        </ul>
+                  <div class="close-btn">&#10005;</div>
+      </div>
```
Comment (FrederickMih) [inline index.html:43]: | Yours | Figma :+1:  |
| --- | --- |
| Screenshot from 2023-03-17 08-51-07 | Screenshot from 2023-03-17 08-55-45 |

- [x] Hi @omer-creator looking at the screenshots above you'll notice that there's a little difference between your design and that of Figma. We need to adjust the font weight of our close-btn and make sure that the vertical distance between the close button and our menu items is the same as that of Figma. You'll kindly required to increase the font weight of the menu items.

File: index.html [hunk 2]
```diff
@@ -30,8 +30,20 @@
...
         </div>
       </nav>
+      <!-- Starting mobile menu  -->
+      <div class="mobile-menu">
+        <div class="mobile-menu-header">
+        </div>
+        <ul class="mobile-menu-items">
+          <li><a href="#portfolio" class="mobile-header-color">Portfolio</a></li>
+          <li><a href="#about" class="mobile-header-color">About</a></li>
+          <li><a href="#contact" class="mobile-header-color">Contact</a></li>
```
Comment (FrederickMih) [inline index.html:40]: - [ ] As stated in the requirements, when the user clicks(taps) any of the mobile menu options, a correct part of the page is displayed. When I click on the #contact option, the contact section was not displayed. Let's kindly consider fixing this issue.

File: Javacsript/mobile-menu.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const menuToggle = document.querySelector('.Sidebar-class');
+const mobileMenu = document.querySelector('.mobile-menu');
+const menuItems = document.querySelectorAll('.mobile-menu-items li a');
+const closeBtn = document.querySelector('.close-btn');
+// const header = document.querySelector('header');
```
Comment (FrederickMih) [inline Javacsript/mobile-menu.js]: - [ ] Let's kindly remove commented codes. We shouldn't commit them to our repo. They make your project look messy. If you need to review the previous version of your code, you can always use git history.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @omer-creator, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- Linters for this project are correctly set up :+1: 



## Required Changes ♻️
**N/A**
_Check the comments under the review._

## Optional suggestions

- This additional information from your README template on the screenshot below at the License header should have been removed in order to render the README file more descriptive. 

Screenshot from 2023-03-16 10-59-40

- Since this project is not deployed, let's kindly remove the **Live Demo** section. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (omer-creator) [reply index.html:43]: Hey, I tried to resolve this issue and it looks same, please confirm it and let me know if it still requires some changes

File: index.html (hunk 2 above)
Comment (omer-creator) [reply index.html:40]: Hey, I have fixed this issue

File: index.html (hunk 1 above)
Comment (codepantha) [reply index.html:43]: - [x] You still have a couple of things to rectify before this gets approved. Kindly increase the `font-size` of your `nav-items` , and also use the correct `margin-left` and `margin-bottom` values in order to be consistent with the Figma design.
- [x] Please notice that you now have horizontal scrolling which wasn't there prior to your most recent commit. Please try to fix it 👍 
### Figma Design
image
### Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @omer-creator 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.



### Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [x] Please consider creating a main branch named `main`. All `feature` branches such as this one `mobile-menu` would be merged into the `main` branch upon approval of your pull request. 
This is important ❗ I implore you to do it 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Yerimah — APPROVED** [review]
## STATUS: APPROVED 🥇 

Hi Team 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f36342e6d656469612e74756d626c722e636f6d2f38376530623634363963393537626638393663646563316361623634366163662f74756d626c725f696e6c696e655f706b316264616b434370317270637239615f3530302e676966

## Highlights
- No linter error ✅ 
- Descriptive PR title and summary ✅ 
- GitHub flow ✅ 
- Functional features ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbeeraTahir — COMMENTED** [review]
# Status: Approved ✔️✔️✔️  

Hi @omer-creator, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

minion

## Highlights
✔️ Linters are passing
✔️ Good implementation of project
✔️ Correct GitHub flow used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — COMMENTED** [review]
Hi @omer-creator, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please make sure to create a new PR and collaborate with their partner. After you are done submit for a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ouasamine — COMMENTED** [review]
Hi @omer-creator 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong PR for this project.

This Activity name is **Portfolio: finish mobile version** and the PR you submitted is about **Mobile Menu** activity, please make sure to submit the correct PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by christianbiring1 — COMMENTED** [review]
Hi @omer-creator , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
Invalid

You have submitted a project, that is not the one you were supposed to submit. I totally understand you have repeated the module, however, always make sure you submit the right PR for a specific milestone as you may notice here you are supposed to submit the PR `finish mobile version` and not the `Add mobile menu` and also always make sure the PR is open and not merged whenever you submit for a re-review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**